tests/test_mechanics.py

```diff
@@ -279,6 +279,31 @@ def test_discard_enchanted_cards():
 	assert not game.player1.hand
 
 
+def test_discover():
```

Comment:
Sounds reasonable. Can you add a TODO to revisit this once we have `hijack` implemented?